go/tasks/plugins/array/catalog.go

```diff
@@ -21,6 +23,8 @@ import (
 	idlCore "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
 )
 
+const awsBatchTaskType = "aws-batch"
```

Comment:
Is there anyway we can define this once and use it everywhere? seems like defining it here and in go/tasks/plugins/array/awsbatch/executor.go makes it more difficult to track.

Reply:
fixed it, thanks.